docs/streams/upgrade-guide.html

```diff
@@ -35,7 +35,7 @@ <h1>Upgrade Guide and API Changes</h1>
 
     <p>
         Upgrading from any older version to {{fullDotVersion}} is possible: you will need to do two rolling bounces, where during the first rolling bounce phase you set the config <code>upgrade.from="older version"</code>
-        (possible values are <code>"0.10.0" - "2.4"</code>) and during the second you remove it. This is required to safely upgrade to the new cooperative rebalancing protocol of the embedded consumer. Note that you will remain using the old eager
+        (possible values are <code>"0.10.0" - "2.3"</code>) and during the second you remove it. This is required to safely upgrade to the new cooperative rebalancing protocol of the embedded consumer. Note that you will remain using the old eager
```

Comment:
@showuon Can we clarify that you only need to do this if you're upgrading from 2.3 or below? I know this seems implied by the fact that the config's possible values stop at 2.3 but there are always creative interpretations of seemingly obvious thing 🙂

Reply:
hi @ableegoldman , After reading the whole paragraph again, I think you're right. 
> This is required to safely upgrade to the new cooperative rebalancing protocol of the embedded consumer. ....  
> you can safely switch over to cooperative at any time once the entire group is on 2.4+ by removing the config value and bouncing.

So, I think here we keep it as `2.3` is fine and correct. 

Thank you.

Reply:
Can we just add a small qualifier in the first line?
`you will need to do two rolling bounces` --> `if upgrading from 2.3 or below, you will need to do two rolling bounces`

Reply:
Thanks for suggestion, @ableegoldman , it makes it more clear! I've updated in this commit https://github.com/apache/kafka/pull/8622/commits/a3accf681e72bbba9a774a464253c1ccd7746188. Thank you.

Reply:
Awesome, thank you!

Reply:
I've also updated in the kafka-site repo: https://github.com/apache/kafka-site/pull/265/commits/513a8205e6b115ca4f876aa5d95d3756061266d5. Thank you.